examples/embedded-graphics-wii/src/main.rs

```diff
@@ -77,7 +76,7 @@ fn main(_argc: isize, _argv: *const *const u8) -> isize {
             area.into_styled(PrimitiveStyle::with_fill(Rgb888::RED))
                 .draw(&mut wii_display)
                 .unwrap();
-            if wii_ctrl.is_button_in_state(Button::A, ButtonState::ButtonHeld) {
+            if wii_ctrl.is_button_held(Button::A) {
```

Comment:
How does is_button_held differ from is_button_down? Does it mean is the button STILL down i.e. it was, the previous check?